mpc-core/src/protocols/rep3.rs

```diff
@@ -463,6 +532,10 @@ impl<F: PrimeField, N: Rep3Network> PrimeFieldMpcProtocol<F> for Rep3Protocol<F,
         dst.b[dst_offset..dst_offset + len].clone_from_slice(&src.b[src_offset..src_offset + len]);
     }
 
+    fn print_share(&self, _: &Self::FieldShare) {
+        panic!("do not print rep3 shares");
+    }
+
```

Comment:
what is the use-case for this method if not to print shares

Reply:
added a print of a